nucliadb_vectors/src/relations/bfs_engine.rs

```diff
@@ -76,55 +81,69 @@ where Guide: BfsGuide
 impl<'a, Guide> BfsEngine<'a, Guide>
 where Guide: BfsGuide
 {
-    pub fn search(mut self) -> RResult<Vec<Entity>> {
-        std::mem::take(&mut self.entry_points)
+    pub fn search(mut self) -> RResult<impl Iterator<Item = GCnx>> {
```

Comment:
Nice improvement in term of user interface :cat:

Reply:
@hermeGarcia :)